providers/gce/gce.go

```diff
@@ -216,6 +218,9 @@ type ConfigGlobal struct {
 	// Default to none.
 	// For example: MyFeatureFlag
 	AlphaFeatures []string `gcfg:"alpha-features"`
+	// ExternalInstanceGroupsPrefix, when not-empty, is used to filter instance groups
+	// and include them in the backend for ILB.
+	ExternalInstanceGroupsPrefix string `gcfg:"external-instance-groups-prefix"`
```

Comment:
I don't like this UX, this makes the solution hard to support because can cause unpredictable behaviors if there is no consistency on prefixes

Reply:
Makes sense. In OpenShift we have a consistent prefix, but there's no guarantee that other setups will.

Open to ideas here; maybe looking for a tag on IGs?

Reply:
I tagged the two main maintainers, they may have some ideas